.reuse/dep5

```diff
@@ -0,0 +1,44 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: talk-desktop
+Upstream-Contact: Nextcloud <info@nextcloud.com>
+Source: https://github.com/nextcloud/talk-desktop/
+
+# L10n
+Files: l10n/*.js l10n/*.json
+Copyright: 2022 Nextcloud GmbH and Nextcloud contributors
+License: AGPL-3.0-or-later
+
+# CI
+Files: .github/**/*
+Copyright: none
+License: CC0-1.0
```

Comment:
Not sure if this is correct or can be just `.github/*`, or must be list of the files like on Takl and Notifications

Reply:
after you updated all workflows half of them should be reported as wrong duplicates, currently it's only one:
```
/home/nickv/.local/pipx/venvs/reuse/lib/python3.11/site-packages/reuse/project.py:286: PendingDeprecationWarning: Copyright and licensing information for '.github/workflows/dependabot-approve-merge.yml' has been found in both '.github/workflows/dependabot-approve-merge.yml' and in the DEP5 file located at '.reuse/dep5'. The information for these two sources has been aggregated. In the future this behaviour will change, and you will need to explicitly enable aggregation. See <https://github.com/fsfe/reuse-tool/issues/779>. You need do nothing yet. Run with `--suppress-deprecation` to hide this warning.
  warnings.warn(
```